content/en/docs/components/trainer/user-guides/pytorch.md

```diff
@@ -4,4 +4,293 @@ description = "How to develop PyTorch models with Kubeflow Trainer"
 weight = 10
 +++
 
-This doc is in progress...
+This page describes how to use `PyTorchJob` for training a machine learning model with [PyTorch](https://pytorch.org/).
+
+The `PyTorchJob` is a Kubernetes
+[custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+to run PyTorch training jobs on Kubernetes. The Kubeflow implementation of
+the `PyTorchJob` is in the [`trainer`](https://github.com/kubeflow/trainer).
+
+## Creating a PyTorch Training Job
```

Comment:
@franciscojavierarceo @kubeflow/wg-training-leads @izuku-sds @astefanutti @Electronic-Waste @varodrig @StefanoFioravanzo @hbelmiro Do we need some sort of introduction to ML Users before this guide ?
For example, what is Kubeflow Trainer Runtime, what is Kubeflow TrainJob from the SDK perspective. 
Maybe we should explain the structure of[ these SDK objects](https://github.com/kubeflow/trainer/blob/master/sdk/kubeflow/trainer/types/types.py#L78-L81) for the ML Users, before we can walk them through Job submission.

I am looking for your suggestions!